### ISSUE TITLE
[Bug report]: appsettings.bundles.json

### ISSUE THREAD

**Issue by klomkling** [serenity-is/Serenity#7290]
### Before submitting the bug report, please read and check the following items

- [X] This isn't a feature request or a discussion topic (Please use [discussions](https://github.com/serenity-is/Serenity/discussions) for that)
- [X] I have searched open and closed issues for duplicates
- [X] I have searched the documentation on the [Serenity Platform website](https://serenity.is/docs/)
- [X] I have read the [frequently asked questions](https://serenity.is/docs/startsharp/faq)
- [X] I have searched my problem in the [discussions and the old issues](https://github.com/serenity-is/Serenity/discussions)

### What happened?

I create new project with command
`dotnet new serene`
everything fine. After `dotnet run` web site is empty page not show anything.
in dev tool console it show error index.global.js not found

### What did you expect to happen?

update project template for appsettings.bundles.json

### How to reproduce?

`dotnet new serene`

### What Serenity Nuget Versions are you seeing the problem on? (separated by comma)

it happened since serene version 8.7.0

### Relevant log output

```console
in appsettings.bundles.json

"~/Serenity.Corelib/index.global.js",
should replace with
"~/Serenity.CoreLib/index.global.js",
```

### Serene template version

8.7.4

### Sergen version

8.7.4

### Code editor

rider

### Operating System

macos

### Node.js version

v23

### TypeScript version

5.4

**Comment by volkanceylan** [comment]
Actually the correct casing is Serenity.Corelib, it changed by mistake to "CoreLib" in https://www.nuget.org/packages/Serenity.Corelib/8.7.3, while it was "Corelib" before https://www.nuget.org/packages/Serenity.Corelib/8.7.2. It should be fixed back to Corelib in 8.8.1.0

**Comment by klomkling** [comment]
@volkanceylan Thanks and waiting for the fixed
